sessions/recap-project-2/character-count-example/js/index.js

```diff
@@ -0,0 +1,15 @@
+console.clear();
+
+const questionElement = document.querySelector('[data-js="question"]');
+const amountLeft = document.querySelector('[data-js="amountLeft"]');
+const maxLength = questionElement.getAttribute('maxLength');
+
+const updateAmountLeft = value => {
+	amountLeft.innerText = value;
+};
+
+updateAmountLeft(maxLength);
+
+questionElement.addEventListener('input', () => {
```

Comment:
Yes! It's right to use the `input` event here. In JS the `change` event is only triggered, when the form field uses focus, while `input` is triggered with each input.

This differs from react's `onChange`, which behaves like the `input` event in vanilla.

https://reactjs.org/docs/dom-elements.html#onchange